src/OpenTelemetry.Instrumentation.MySqlData/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 * Update OTel API version to `1.3.1`.
   ([#631](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/631))
+* Compatibility with `Mysql.Data` 8.0.31 or later, Users must set `Logging=true`
```

Comment:
I think that it should be mentioned also in README.md

Reply:
You are right, I added in [`a545f4a` (#692)](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/692/commits/a545f4a38ffe3090810e977863ce85c33d108151)